### ISSUE TITLE
Snow on Tiled Roofs

### ISSUE THREAD

**Issue by Zygus42** [ldtteam/Structurize#72]
minecolonies-1.12.2-0.10.351-ALPHA-universal

### Expected behavior

Snow fitting to roof tile shape.

### Actual behaviour

![2019-09-25_20 18 14](https://user-images.githubusercontent.com/20757445/65601348-83466400-dfd4-11e9-8d1b-7f76555fd6e5.png)

### Steps to reproduce the problem

Let it snow.

**Comment by Asherslab** [comment]
there's nothing we can generally do about that other than simply disabling snow on our roof tiles

**Comment by Raycoms** [comment]
Theoretically we could make our own version of "snow" which is smoother on
it. I think there is stuff for that probably.

Am Fr., 27. Sept. 2019 um 06:28 Uhr schrieb AshersLab <
notifications@github.com>:

**Comment by Raycoms** [comment]
No possibility for an overlay or something like that? If not, we can just
set that snow can't land on any of our shingles or shingle slabs.

Am Fr., 27. Sept. 2019 um 06:32 Uhr schrieb AshersLab <
notifications@github.com>:

**Comment by Asherslab** [comment]
an overlay? not sure what you mean there, any rendering is done via models.

**Comment by Raycoms** [comment]
I don't know, something like what is done with the waterlogged stuff of
blocks.

Am Fr., 27. Sept. 2019 um 06:57 Uhr schrieb AshersLab <
notifications@github.com>:

**Comment by cltnschlosser** [comment]
Could you just use a different texture?

Colton Schlosser

**Comment by Asherslab** [comment]
@cltnschlosser would require a different model (as mentioned)
@Raycoms waterlogged is a full block of water "overlayed"
while that would technically work, that's not how snow layers usually look in MC, it would be really weird looking that way, plus it wouldn't be solid, we'd also have to create a crap tonne of code to mimic that vanilla behaviour (waterlogged blocks changed crap tonnes of stuff in mc)

**Comment by Raycoms** [comment]
OKay, then we should probably just disable snow on them

**Comment by Zygus42** [comment]
There's several mods that add snow layers to stairs etc, but I tried them all when i was building my latest mod pack, and they use weird workarounds that are not very server friendly.

**Comment by Asherslab** [comment]
@Zygus42 yes there are mods that enable that type of thing, however that would require us to also add code to either support those mods, or add those types of features into our own, this isn't something that we'd really want to do, nor would it be very feasible due to the fact that our shingles, while similar to stairs and technically are stairs, have different models/textures/etc from stairs

**Comment by Zygus42** [comment]
I wasn't suggesting you do, in fact, I was just pointing out that there doesn't seem an easy solution to this problem in minecraft as it stands.  Without, as you say, adding new models for a fairly fringe situation.  On the other hand, snow would fit the look of your tile set perfectly, making it look like a european christmas scene.
